chore: add docz (#3)
leroy-merlin-br/backyard-ui: docs/index.mdx

---
name: Home
route: /
---

# Backyard UI

Created based on Leroy Merlin Brazil needs, and used internally as well.

> It is for [**React**](https://reactjs.org) apps and follows [**Standard Style Guide**](https://github.com/standard/standard).

Reviewer: ```suggestion
It is designed to work with [**React**](https://reactjs.org) apps and it follows the [**Standard Style Guide**](https://github.com/standard/standard).
```

Fully focused library for developers and UX designers, always thinking about productivity gain.

## Useful links

> To help you get started we leave below some links that may be useful for you:

Author: Vou procurar uma outra forma de formular essa frase então.

> To help you get started we leave below some links that may be useful for you:

Reviewer: ```suggestion
To learn more about **Backyard UI** you might want to check the resources below:
```

> - [Getting Started](/getting-started) - Steps how to start using library.

Reviewer: ```suggestion
- [Getting Started](/getting-started) - Steps on how to start using library.
```

> - [Documenting](/documenting) - Instructions for create documentation a new component.

Reviewer 2: ```suggestion
- [Documenting](/documenting) - How to create a component documentation
```

nao sei.

> - [Documenting](/documenting) - Instructions for create documentation a new component.

Reviewer: ```suggestion
- [Documenting](/documenting) - Instructions to document a new component.
```

> - [Contribute](/contribute) - Instructions for people considering contribution to Backyard.

Reviewer 2: ```suggestion
- [Contribute](/contribute) - How to contribute with Backyar.
```

> - [Contribute](/contribute) - Instructions for people considering contribution to Backyard.

Reviewer: ```suggestion
- [Contribute](/contribute) - Instructions for people considering to contribute to Backyard UI.
```
